plugins/hotfix.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: hotfix
```

Comment:
`hotfix` is not very descriptive. I think your name should mention `sync` or `transfer`, like your short description does.